### ISSUE TITLE
AttributeError: module 'warnings' has no attribute 'UserWarning' when installing package with multi-line description

### ISSUE THREAD

**Issue by greschd** [pypa/setuptools#2539]
When installing a package with a multi-line description with the latest setuptools version (51.3.2), I get the following error:

```python
      File "/home/greschd/.virtualenvs/tmp-9b3743628834a12/lib/python3.8/site-packages/setuptools/dist.py", line 125, in single_line
        warnings.UserWarning("newlines not allowed and will break in the future")
    AttributeError: module 'warnings' has no attribute 'UserWarning'
```

This seems to be related to the discussion in https://github.com/pypa/setuptools/issues/1390, and specifically the change in https://github.com/pypa/setuptools/pull/2538, because `UserWarning` is a built-in global, not part of the `warnings` module:

```python
>>> import warnings
>>> warnings.UserWarning
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'warnings' has no attribute 'UserWarning'
>>> UserWarning
<class 'UserWarning'>
>>> 
```

The fix should be very straightforward, using `warnings.warn` instead.

**Comment by jaraco** [comment]
Yeah, that was dumb. My mistake for not testing it at all. I'll push out a fix right away.

**Comment by jaraco** [comment]
Should be fixed in v51.3.3.

**Comment by greschd** [comment]
Thanks for the quick response, and all the good work on setuptools! :rocket:
